api/releases.go

```diff
@@ -289,6 +290,21 @@ func releaseDeployHandler(c echo.Context) error {
 		if err != nil {
 			log.WithError(err).Error("failed release rollout")
 		}
+		if config.GetString(config.CiProvider) != "" {
+			rolloutFailed := false
+			for ix := range release.Waves {
+				releaseRolloutWave := releaseRollout.Waves[ix]
+				if releaseRolloutWave.Status == types.RolloutStatusFailed {
+					rolloutFailed = true
+					log.Errorf("Rollout status: %+v ", releaseRolloutWave.Status)
+					break
+				}
+			}
+			if rolloutFailed == false {
+				InitializeCiClient(config.GetString(config.CiProvider))
```

Comment:
This should probably be initialized only once, and somewhere other than this deployment stage.

Reply:
updated and moved initialization to vili.go